server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/test/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultRoutesTest.java

```diff
@@ -198,7 +199,7 @@ void beforeEach() throws Exception {
         taskManager = new MemoryTaskManager(new Hostname("foo"));
         JsonTransformer jsonTransformer = new JsonTransformer();
 
-        RestoreService vaultRestore = new RestoreService(vault, mailboxManager);
+        RestoreService vaultRestore = new RestoreService(vault, mailboxManager, RestoreLocationConfiguration.DEFAULT);
```

Comment:
Shouldn't we merge `RestoreLocation` and `RetentionConfiguration` pojos into a single DeletedMessageVaultConfiguration pojo with 2 fields?

Reply:
I was keeping them separate since they are used in different locations in the code base. And initially did not want to introduce more change impact than necessary, while it all was a private patch. But of course I can merge it all together now, which will also eliminate the extra module you commented on above.

Reply:
I am globally ok with the patch as off today but would prefer a single pojo ;-)

Reply:
Done, see latest push.